src/routes/safe/components/Settings/ManageOwners/ReplaceOwnerModal/screens/Review/index.tsx

```diff
@@ -174,17 +174,17 @@ export const ReviewReplaceOwnerModal = ({
                 </Row>
                 <Hairline />
                 {owners?.map(
-                  (owner) =>
-                    owner.address !== ownerAddress && (
-                      <React.Fragment key={owner.address}>
+                  (safeOwner) =>
+                    sameAddress(safeOwner.address, owner.address) && (
```

Comment:
Same case here. I'm I understanding something wron?

Reply:
fixed, thank you Dani 🙏